5.sc_Morphology_Logistic_Regression/0.split_data/0.split_data_binary_classification.ipynb

```diff
@@ -0,0 +1,851 @@
+{
```

Comment:
<p>Same comment here about DRYness in the same analysis</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>